### ISSUE TITLE
Multiple `Get-ChildNode -Include` and `-Exclude` pattern don't work

### ISSUE THREAD

**Issue by iRon7** [iRon7/ObjectGraphTools#36]
```PowerShell
        $Object = @{
            Comment = 'Sample ObjectGraph'
            Data = @(
                @{
                    Index = 1
                    Name = 'One'
                    Comment = 'First item'
                }
                @{
                    Index = 2
                    Name = 'Two'
                    Comment = 'Second item'
                }
                @{
                    Index = 3
                    Name = 'Three'
                    Comment = 'Third item'
                }
            )
        }
```

Single selections work as expected:
```PowerShell
$Object | Get-ChildNode -Recurse -Include Comment

PathName         Name    Depth Value
--------         ----    ----- -----
.Data[0].Comment Comment     3 First item        
.Data[1].Comment Comment     3 Two to the Loo    
.Data[2].Comment Comment     3 Third item        
.Comment         Comment     1 Sample ObjectGraph
```

```PowerShell
$Object | Get-ChildNode -Recurse -Exclude Comment

PathName       Name  Depth Value
--------       ----  ----- -----
.Data          Data      1 {System.Collections.Hashtable, System.Collections.Hashtable, System.Collections.Hashtable}
.Data[0].Name  Name      3 One
.Data[0].Index Index     3 1
.Data[1].Name  Name      3 Two
.Data[1].Index Index     3 2
.Data[2].Name  Name      3 Three
.Data[2].Index Index     3 3
```

And a `-Literal` selection works for multiple names:
```PowerShell
$Object | Get-ChildNode -Recurse -Include Comment, Name -Literal

PathName         Name    Depth Value
--------         ----    ----- -----
.Data[0].Comment Comment     3 First item
.Data[0].Name    Name        3 One
.Data[1].Comment Comment     3 Two to the Loo
.Data[1].Name    Name        3 Two
.Data[2].Comment Comment     3 Third item
.Data[2].Name    Name        3 Three
.Comment         Comment     1 Sample ObjectGraph
```

```PowerShell
$Object | Get-ChildNode -Recurse -Exclude Comment, Name -Literal

PathName       Name  Depth Value
--------       ----  ----- -----
.Data          Data      1 {System.Collections.Hashtable, System.Collections.Hashtable, System.Collections.Hashtable}
.Data[0].Index Index     3 1
.Data[1].Index Index     3 2
.Data[2].Index Index     3 3
```

But the result is incorrect for multiple wildcard selections:
```PowerShell
$Object | Get-ChildNode -Recurse -Include Comment, Name

PathName         Name    Depth Value
--------         ----    ----- -----
.Data            Data        1 {System.Collections.Hashtable, System.Collections.Hashtable, System.Collections.Hashtable}
.Data[0].Comment Comment     3 First item
.Data[0].Name    Name        3 One
.Data[0].Index   Index       3 1
.Data[1].Comment Comment     3 Two to the Loo
.Data[1].Name    Name        3 Two
.Data[1].Index   Index       3 2
.Data[2].Comment Comment     3 Third item
.Data[2].Name    Name        3 Three
.Data[2].Index   Index       3 3
.Comment         Comment     1 Sample ObjectGraph
```

```PowerShell
$Object | Get-ChildNode -Recurse -Exclude Comment, Name

# Returns nothing
```

**Expected** the specific wildcard selection to be the same as the literal selection

**Comment by iRon7** [comment]
Fixed in `0.0.14`:

```PowerShell
$Object | Get-ChildNode -Recurse -Include Com*, Nam?

PathName         Name    Depth Value
--------         ----    ----- -----
.Data[0].Comment Comment     3 First item
.Data[0].Name    Name        3 One
.Data[1].Comment Comment     3 Second item
.Data[1].Name    Name        3 Two
.Data[2].Comment Comment     3 Third item
.Data[2].Name    Name        3 Three
.Comment         Comment     1 Sample ObjectGraph
```

```PowerShell
$Object | Get-ChildNode -Recurse -Exclude Com*, Nam?

PathName       Name  Depth Value
--------       ----  ----- -----
.Data          Data      1 {System.Collections.Hashtable, System.Collections.Hashtable, System.Collections.Hashtable}
.Data[0].Index Index     3 1
.Data[1].Index Index     3 2
.Data[2].Index Index     3 3
```

**Comment by iRon7** [comment]
fixed 0.0.14
